core/src/main/java/hudson/model/PeriodicWork.java

```diff
@@ -107,12 +107,12 @@ public static void init() {
         ExtensionList<PeriodicWork> extensionList = all();
         extensionList.addListener(new PeriodicWorkExtensionListListener(extensionList));
         for (PeriodicWork p : extensionList) {
-            schedulePeriodicWork(p);
+            p.schedulePeriodicWork();
         }
     }
 
-    private static void schedulePeriodicWork(PeriodicWork p) {
-        Timer.get().scheduleAtFixedRate(p, p.getInitialDelay(), p.getRecurrencePeriod(), TimeUnit.MILLISECONDS);
+    protected void schedulePeriodicWork() {
```

Comment:
Needs Javadoc and `@since`.

Reply:
Done via 7faed41d1df856493cd3157ce79db98ca2521485